flutter/test/sentry_asset_bundle_test.dart

```diff
@@ -333,6 +333,159 @@ void main() {
       },
     );
 
+    // TODO: REMOVE ME
```

Comment:
@arnemolland Should we remove the TODO or the test?

Reply:
The TODO was my own note, removing.